akka-actor/src/main/scala/akka/pattern/BackoffSupervisor.scala

```diff
@@ -291,10 +292,16 @@ object BackoffSupervisor {
 
   final case class RestartCount(count: Int)
 
-  // not final for binary compatibility with 2.4.1
+  /**
+   * INTERNAL API
+   */
+  @InternalApi
   private[akka] case object StartChild extends DeadLetterSuppression
 
-  // not final for binary compatibility with 2.4.1
```

Comment:
I thought this a worthwhile note

Reply:
It's internal now (not sure if it ever wasn't) so we don't need to keep bincomp.